pipelines/batch/src/test/java/com/google/fhir/analytics/FhirSearchUtilTest.java

```diff
@@ -111,6 +113,7 @@ public void testFindBaseSearchUrl() {
   @Test
   public void testSearchForResource() {
     Bundle actualBundle = fhirSearchUtil.searchByUrl(SEARCH_URL, 10, SummaryEnum.DATA);
+    Assert.assertNotNull(actualBundle);
```

Comment:
For consistency, please use `hamcrest.Matchers`  for null checks, e.g., see `notNullValue` and `nullValue` usage in this file.